rancher/v1.4/en/cattle/secrets/index.md

```diff
@@ -0,0 +1,113 @@
+
+### Creating Secrets
+
+Secrets are created and scoped at an environment level, which means there can only be one secret with the same name in an environment. Any container in the same environment can share the same secrets. For example, a database password, i.e. `db_password`, can be used in a database container and a Wordpress container. 
+
+> **Note:** Rancher CLI currently does not support adding secrets to Rancher. 
+
+Go to **Infrastructure** -> **Secrets**. Provide a **Name** and a **Secret Value** and **Save** the secret. 
+
+Once a secret is created, the secret value **cannot** be edited or updated. If you need to change an existing value of a secret, the only way to do so is to delete the secret. After a secret has been updated, any services using the secret will need to be re-launched with the updated secret. 
+
+### Deleting Secrets
+
+In the UI, secrets can be deleted from Rancher, but it does not remove the secret (i.e. file) from any container using the secret or on the host that are running containers using a secret. 
+
+### Enabling Secrets in Containers
+
+In order to consume secrets in containers, the `Rancher Secrets` service will need to be launched. This service can be deployed by either adding it to [environment templates]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#what-is-an-environment-template) so that it's deployed on all [environments]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/) or by launching it directly from the [Rancher Catalog]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/catalog/). If you are adding this service to an existing environment, navigate to **Catalog** -> **Library** and search for the `Rancher Secrets` entry. Without launching this catalog entry, you will only be able create secrets, but not be able to use them in your containers.
+
+### Adding Secrets to Services/Containers 
+
+Secrets can be added into a service/container under the **Secrets** tab during service/container creation. 
+
+* Select **Add Secret**
+* The list of available secrets that have been created in Rancher are in a drop down menu. Select the secret that you'd like to use. 
+* (Optional) Input a different filename for the secrets file. By default, it will use the name of the secret for the filename. 
+* (Optional) If the default file mode needs to be modified, click the **Customize file ownership & permissions** link. The User ID, Group ID and File Mode (octal) can be updated. By default, the User ID is `0`, the Group ID is `0` and the file mode is `0444`.
+
+Click **Create**. 
+
+When secrets are added to a container, the secrets are written to two places.
+
+* Inside the container: A tmpfs volume is mounted at `/run/secrets/`. 
+* On the host that is running the container using the secrets: A `rancher-secret` volume is mounted on the host at `/var/lib/rancher/volumes/rancher-secrets/`.
```

Comment:
Its really the same tmpfs volume, it can be seen from both the host and the container via the two paths.